src/core/qgsvectordataprovider.h

```diff
@@ -524,6 +524,16 @@ class CORE_EXPORT QgsVectorDataProvider : public QgsDataProvider, public QgsFeat
      */
     virtual bool hasMetadata() const { return true; };
 
+    /**
+     * Listen to provider notification and transforms them
```

Comment:
The dox should mention explicitly when something is a setter, e.g. "Sets whether the provider will listen to provider notifications..."